modules/ROOT/nav.adoc

```diff
@@ -48,6 +51,7 @@
 **** xref:dotnet-projects.adoc[.NET projects]
 **** xref:bower-projects.adoc[Bower projects]
 **** xref:zip-install.adoc[TinyMCE .zip deployments]
+** xref:integrations.adoc[TinyMCE Integrations]
```

Comment:
This was specifically removed/moved to the installation section from what I recall as this was part of Content Strategy changes for 6.0. I don't think we should have this, but we should check with Brian.

Reply:
Yeah, there are parts of the docs that link to see integrations, which is very generic, so I put this placeholder in.  It would be just a simple integrations index that filters out to specific integrations